### PR TITLE
Create Venmo Module Shell

### PR DESCRIPTION
**Pull request by agedd**
### Summary of changes

- Create `VenmoPayments` and `VenmoPaymentsTests` schemes as well as appropriate targets 
- Create `VenmoPayments` directory in `Sources` and `VenmoPaymentsTests` directory in `UnitTests`
- Add `VenmoPayments` module definition to `PayPal.podspec` and `Package.swift`

### Checklist

- [x] Added a changelog entry

- @agedd

### CODE REVIEW

File: PayPal.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
   "pins" : [
     {
       "identity" : "paypalcheckout-ios",
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/paypal/paypalcheckout-ios",
       "state" : {
-        "revision" : "f477176da4c6780c8586b493c604f0ab74d9be49",
-        "version" : "1.2.0"
+        "revision" : "da1b998dcab91011b05df139a6819cc58dd45ccd",
+        "version" : "1.3.0"
```
Comment (jaxdesmarais) [inline PayPal.xcworkspace/xcshareddata/swiftpm/Package.resolved]: We will want to revert the changes in this file

File: PayPal.podspec [hunk 2]
```diff
@@ -39,6 +39,11 @@ Pod::Spec.new do |s|
     s.resource_bundle = { "PayPalWebPayments_PrivacyInfo" => "Sources/PayPalWebPayments/PrivacyInfo.xcprivacy" }
   end
 
+  s.subspec "VenmoPayments" do |s|
+    s.source_files  = "Sources/VenmoPayments/*.swift"
```
Comment (jaxdesmarais) [inline PayPal.podspec:43]: Looking at the files it seems like the `VenmoPayments` directory is not currently in `Sources`

Comment (agedd) [reply PayPal.podspec:43]: addressed in b1a40ac05164a7165d03e91f862683de837141ca

Comment (agedd) [reply PayPal.podspec:43]: tyyy, addressed in b1a40ac05164a7165d03e91f862683de837141ca

File: Package.swift [hunk 3]
```diff
@@ -31,6 +31,10 @@ let package = Package(
         .library(
             name: "FraudProtection",
             targets: ["FraudProtection", "PPRiskMagnes"]
+        ),
+        .library(
```
Comment (KunJeongPark) [inline Package.swift]: nit pick: do we want to move this above the Magnes library?
Looks like all the payment method libraries are together and then Magnes.
Otherwise it looks good!

**Review by KunJeongPark — APPROVED** [review]

File: Package.swift (hunk 3 above)
Comment (KunJeongPark) [reply Package.swift]: Looks like you have to resolve this:
/Users/runner/work/paypal-ios/paypal-ios/Sources/VenmoPayments/VenmoClient.swift:4:1: error: Vertical Whitespace before Closing Braces Violation: Don't include vertical whitespace (empty line) before closing braces (vertical_whitespace_closing_braces)

File: Sources/VenmoPayments/VenmoClient.swift [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+public class VenmoClient: NSObject {
```
Comment (scannillo) [inline Sources/VenmoPayments/VenmoClient.swift]: Nit we don't need ObjC support for PPCP

```suggestion
public class VenmoClient {
```

File: Sources/VenmoPayments/VenmoClient.swift [hunk 5]
```diff
@@ -0,0 +1,4 @@
+import Foundation
+
+public class VenmoClient {
+}
```
Comment (jaxdesmarais) [inline Sources/VenmoPayments/VenmoClient.swift]:
```suggestion
public class VenmoClient {
    // TODO: to be implemented in a future PR
}
```

File: UnitTests/VenmoPaymentsTests/VenmoClient_Tests.swift [hunk 6]
```diff
@@ -0,0 +1,5 @@
+import XCTest
+
+public class VenmoClient_Tests: XCTestCase {
+
```
Comment (jaxdesmarais) [inline UnitTests/VenmoPaymentsTests/VenmoClient_Tests.swift]:
```suggestion
public class VenmoClient_Tests: XCTestCase {
    // TODO: to be implemented in a future PR
```

**Review by stechiu — APPROVED** [review]

**Review by richherrera — APPROVED** [review]
LGTM!!! 🚀
